packages/babel-plugin/src/types.ts

```diff
@@ -103,6 +103,12 @@ export interface PluginOptions {
    * Defaults to `true`.
    */
   sortAtRules?: boolean;
+
+  /**
+   * Adds a defined prefix to the generated classes' hashes.
+   * Useful in micro frontend environments to avoid clashing/specificity issues.
+   */
+  hashPrefix?: string;
```

Comment:
nit

what about `addClassPrefix`? cc @dddlr @JakeLane

Reply:
yeah more descriptive naming is helpful

Reply:
Imo `add` suggests the expected value is a boolean and not a string that we're setting. Would `classHashPrefix` be better in this case?

Reply:
ye that's fine